packages/nextjs/src/import-map/default-import-map.ts

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import {
+  Link,
+  Text,
+  RichText,
+  useSitecore,
+  NextImage,
+  Placeholder,
+  Image,
+  File,
+  useComponentProps,
+  withDatasourceCheck,
+  CdpHelper,
+  withSitecore,
+} from '..';
+
+export interface ImportEntry {
+  module: string;
+  exports: { name: string | 'default' | '*'; value: unknown }[];
+}
+
+export const importEntries: ImportEntry[] = [
```

Comment:
lets name it as `defaultImportEntires` so when it is imported in starter kit code it will look like
`import { defaultImportEntires } from '@sitecore-content-sdk/next/import-map'